src/metaxy/migrations/detector.py

```diff
@@ -155,6 +155,8 @@ def detect_migration(
 
     yaml_path = migrations_dir / f"{migration_id}.yaml"
     migration_yaml = {
+        "migration_type": migration.migration_type,
```

Comment:
What's these changes about?

Reply:
was required after the rebase from main to make sure the tests do not fail

Reply:
actually not only for this branch here